nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/core/im/IMImageUtils.java

```diff
@@ -123,6 +123,11 @@ protected void makeFiles(Blob blob, String targetExt)
                 targetExt = ext;
             }
             targetFile = Framework.createTempFile("nuxeoImageTarget", "." + targetExt);
+
+            Framework.trackFile(targetFile, targetFile);
+            if (tmpFile != null) {
+                Framework.trackFile(tmpFile, tmpFile);
+            }
```

Comment:
Looking quickly at the code I'm wondering:
- If `sourceFile ` shoudn't be tracked?
- Why we track `tmpFile` as it seems to be handled [here](https://github.com/nuxeo/nuxeo/blob/10.10/nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/core/im/IMImageUtils.java#L79)?

Reply:
yep, thx :):

-  _If sourceFile shoudn't be tracked?_: it's what id did initially but after that the tests will failed if we launch them a second time, because `sourceFile` are those embedded in the Nuxeo source.

-  _Why we track tmpFile as it seems to be handled here?_:  I added it for the `makeFiles` method [here](https://github.com/nuxeo/nuxeo/blob/b706b4740d6169c541d474a6a2733d52a21c63fb/nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/core/im/IMImageUtils.java#L88) before this fix in this method we didn't track any file but we did it on `call` the once you give [here](https://github.com/nuxeo/nuxeo/blob/b706b4740d6169c541d474a6a2733d52a21c63fb/nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/core/im/IMImageUtils.java#L57)